package.json

```diff
@@ -68,10 +68,14 @@
   },
   "devDependencies": {
     "@applitools/eyes-cypress": "^3.23.4",
+    "@babel/core": "^7.16.5",
```

Comment:
All the required babel tools should be already included with `next` (except `@babel/eslint-parser` maybe)

Reply:
Needed to keep the parser, but everything else removed